### ISSUE TITLE
Multiple PDR peaks doesn't work

### ISSUE THREAD

**Issue by vetlewi** [oslocyclotronlab/Talys-code#2]
In the documentation, TALYS supports up to two PDR peaks. However, TALYS will only use the first peak.

**Comment by cecgresonant** [comment]
That's a bad bug.. Did you write to the Stephane's about it?

**Comment by vetlewi** [comment]
> That's a bad bug.. Did you write to the Stephane's about it?

No, not yet. But I have a fix, see 1ec60de71c78a74cca4e5bae402b157f6558d88f

In addition, I'd like to add a change in the code where TALYS writes the output file to ensure that all PDR parameters are properly written. This will require:
* Add a new global variable similar to the `ngr` but for PDR
* Modify `gammapar.f` to count the number of PDRs for each multipolarity (similar to how it handles `ngr`)
* Modify `gammaout.f` to write PDR parameters to the output
* Modify `partable.f` to include the PDR parameters in the model parameters written to file

**Comment by vetlewi** [comment]
To be clear, the issue is as following. If you add two E1 PDR to the strength function of a given nucleus, say 145Nd you'd write in the input to TALYS

```
epr 60 145 7.5 E1 1
spr 60 145 1.5 E1 1
gpr 60 145 0.9 E1 1

epr 60 145 9.5 E1 2
spr 60 145 12.2 E1 2
gpr 60 145 1.2 E1 2
```
And you'd expect there to be two standard Lorentzians added to the E1 strength with the given parameters. However, TALYS only uses the first set of parameters (the ones marked with 1), thus only one extra Lorentzian is added to the E1 strength.
